tests/Tests.elm

```diff
@@ -993,5 +993,30 @@ all =
                         0
                         (List.range 1 1000)
                         |> Expect.equal 55
+            , test "after Stop, the function is not called anymore" <|
+                let
+                    throwRangeErrorException : () -> a
+                    throwRangeErrorException () =
+                        preventTailCallOptimization ()
+
+                    preventTailCallOptimization : () -> a
+                    preventTailCallOptimization () =
+                        throwRangeErrorException ()
+                in
+                \() ->
+                    stoppableFoldl
+                        (\n acc ->
+                            if n < 50 then
+                                Continue n
+
+                            else if n == 50 then
+                                Stop n
+
+                            else
+                                explode ()
```

Comment:
@Chadtech I've just realized this probably is missing a function name change 😰  `explode` can't be found anywhere, if my memory's correct it was supposed to be `throwRangeErrorException ()` instead of `explode ()`. This will likely not compile.

Sorry, I guess a commit got lost / didn't get pushed / ... I'll create a fix PR

Reply:
No worries! I noticed the code wasn't compiling after I merged it in, but it was an easy fix.

Thanks for your original contribution, and thanks for noticing now!